build.gradle

```diff
@@ -203,15 +211,25 @@ dependencies {
   implementation group: 'org.springframework.boot', name: 'spring-boot-starter-actuator'
   implementation group: 'org.springframework.boot', name: 'spring-boot-starter-aop'
   implementation group: 'org.springframework.boot', name: 'spring-boot-starter-json'
+  implementation group: 'org.springframework.boot', name: 'spring-boot-starter-data-jpa'
```

Comment:
Is this dependency needed?  The commissioning service doesn't use a database so this may be able to be removed.

Reply:
removed